test/commands/BeautifyCommand.spec.ts

```diff
@@ -75,8 +74,40 @@ describe("BeautifyCommand", () => {
           expect(json.stdout).toBe('const test = "test";\n');
         });
     });
+    test("should beautify using stdin with TTY", async () => {
+      const command = new CustomCommand("const test = 'test';", true);
+      return command
+        .beautify({
+          args: [],
+          configFile: "test/.unibeautifyrc.yml",
+          language: "JavaScript",
+        })
+        .then(() => {
+          const json = command.toJSON();
+          expect(json.exitCode).toBe(0);
+          expect(json.stderr).toBe("");
+          // tslint:disable-next-line:quotemark
+          // expect(json.stdout).toBe('const test = "test";\n'); // <--- Doesn't work
```

Comment:
Remove this commented out code.

Do we not have a linting rule checking this? 🔥 https://github.com/Unibeautify/unibeautify-cli/blob/master/tslint.json Nope! Will need to add https://github.com/Glavin001/tslint-clean-code#supported-rules with `no-commented-out-code` rule.